packages/opencensus-core/src/stats/model/aggregation.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Aggregation, BucketBoundaries, Histogram, Measurement} from './types';
+
+export class StatsAggregation implements Aggregation {
+  private bucketBoundaries: BucketBoundaries;
+  private measurements: Measurement[];
+
+  constructor(measurements: Measurement[], bucketBoundaries: BucketBoundaries) {
+    this.measurements = measurements;
+    this.bucketBoundaries = bucketBoundaries;
+  }
+
+  /**
+   * Counts the number of recorded measurements.
+   */
+  count(): number {
+    return this.measurements.length;
+  }
+
+  /**
+   * Returns the recorded measurements values summed.
+   */
+  sum(): number {
+    let sum = 0;
+    for (const measurement of this.measurements) {
+      sum += measurement.value;
+    }
+    return sum;
+  }
+
+  /**
+   * Returns the last recorded measurement value.
+   */
+  lastValue(): number {
+    return this.measurements[this.count() - 1].value;
+  }
+
+  /**
+   * Returns a histogram distribution. A distribution Aggregation may contain a
+   * histogram of the values in the population.
+   */
+  distribution(): Histogram[] {
```

Comment:
Is this meant to be cumulative? If so, what is the point of `value1`?

Reply:
Value1 and value2 were boundaries of measurements values. Anyway, I changed to min and max, I think it's better to understand.